{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cve@kernel.org",
			"ID": "CVE-2021-47039"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "In the Linux kernel, the following vulnerability has been resolved:\n\nataflop: potential out of bounds in do_format()\n\nThe function uses \"type\" as an array index:\n\n\tq = unit[drive].disk[type]->queue;\n\nUnfortunately the bounds check on \"type\" isn't done until later in the\nfunction.  Fix this by moving the bounds check to the start."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://git.kernel.org/stable/c/07f86aa8f4fe077be1b018cc177eb8c6573e5671",
					"url": "https://git.kernel.org/stable/c/07f86aa8f4fe077be1b018cc177eb8c6573e5671"
				},
				{
					"name": "https://git.kernel.org/stable/c/07f86aa8f4fe077be1b018cc177eb8c6573e5671",
					"url": "https://git.kernel.org/stable/c/07f86aa8f4fe077be1b018cc177eb8c6573e5671"
				},
				{
					"name": "https://git.kernel.org/stable/c/1ffec389a6431782a8a28805830b6fae9bf00af1",
					"url": "https://git.kernel.org/stable/c/1ffec389a6431782a8a28805830b6fae9bf00af1"
				},
				{
					"name": "https://git.kernel.org/stable/c/1ffec389a6431782a8a28805830b6fae9bf00af1",
					"url": "https://git.kernel.org/stable/c/1ffec389a6431782a8a28805830b6fae9bf00af1"
				},
				{
					"name": "https://git.kernel.org/stable/c/2a3a8bbca28b899806844c00d49ed1b7ccb50957",
					"url": "https://git.kernel.org/stable/c/2a3a8bbca28b899806844c00d49ed1b7ccb50957"
				},
				{
					"name": "https://git.kernel.org/stable/c/2a3a8bbca28b899806844c00d49ed1b7ccb50957",
					"url": "https://git.kernel.org/stable/c/2a3a8bbca28b899806844c00d49ed1b7ccb50957"
				}
			]
		}
	},
	"impact": {},
	"configurations": {
		"CVE_data_version": "4.0"
	},
	"lastModifiedDate": "2024-11-21T06:35Z",
	"publishedDate": "2024-02-28T09:15Z"
}
